browser-extension/styles.css

```
@@ -0,0 +1,66 @@
+  justify-content: center;
+  width: 34px;
+  height: 34px;
+  border-radius: 9999px;
+  transition: background-color 120ms ease, color 120ms ease;
+}
+
+.nostr-competency-action-svg {
+  width: 18px;
+  height: 18px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)